src/avatar.h

```diff
@@ -211,4 +215,95 @@ class avatar : public player
         int per_upgrade = 0;
 };
 
+struct points_left {
```

Comment:
Does all of this have to be in the header? At least the functions calling `get_option` should be in the cpp file, so the header does not have to include "options.h".